RELEASE_NOTES.md

```diff
@@ -0,0 +1,17 @@
+Release 1.1.1 of the Amazon Kinesis Video C Producer SDK
```

Comment:
3.1.1

Reply:
That's for CPP, 1.1.1 is correct for C Producer.

Reply:
D’oh